### ISSUE TITLE
slow rendering in fill graph

### ISSUE THREAD

**Issue by MySchizoBuddy** [rstudio/dygraphs#65]
My graph has 8760 points and with fillGraph option it is slow to zoom and select range with range selector option.
This is fixed in dygraphs.js 1.1.0 version. Becareful though dygraph 1.1.0 has shrunk it's size and moved code to an addition "extra" code

**Comment by jjallaire** [comment]
Yes, I tried to update to dygraphs 1.1.0 once and the axes labels started
to behave strangely. There were a couple of users that wanted to see the
update and I asked if they could sort out why the labels were having
problems but that never went anywhere.
